VAGOV-2855: Display 'Other VA locations' via Facility ID

aurora-a-k-a-lightning opened the pull request:
## Description
Displays "Other VA Locations" on the locations page of a facility

I decided to add another react widget because of how often the designs change - I want the flexibility from the get-go since the design is in flux.

## Testing done
Locally with staging and dev data
Pre-req: May have to make sure there is data in the Pittsburgh Health Care system for "Other VA Locations" to populate `fieldOtherVaLocations` http://staging.cms.va.gov/node/318/edit?destination=/admin/content

You can fill in `nca_120` for the soldier lot that is in the mock ups

1. re-build site with either staging or dev data
2. browse to http://localhost:3001/pittsburgh-health-care/locations/
3. scroll all the way down to see the "Other VA Locations" section

## Screenshots
![localhost_3001_pittsburgh-health-care_locations_ (2)](https://user-images.githubusercontent.com/19178435/58582737-52614880-8207-11e9-9501-69df5de7423e.png)

## Acceptance criteria
https://va-gov.atlassian.net/browse/VAGOV-2855

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

aurora-a-k-a-lightning committed:
VAGOV-3243

- change “Main location” headline to “Main locations(s)”

- remove padding and grey container for location listings

- remove `Location details >` link from results in “Our locations” list

aurora-a-k-a-lightning committed:
VAGOV-3243: fix linting

aurora-a-k-a-lightning committed:
VAGOV-2855: add "Other VA locations" react widget

aurora-a-k-a-lightning committed:
VAGOV-2855: add unit test

aurora-a-k-a-lightning committed:
Merge branch 'master' of https://github.com/department-of-veterans-affairs/vets-website into vagov-2855

aurora-a-k-a-lightning committed:
VAGOV-2855: finish up unit tests

aurora-a-k-a-lightning committed:
VAGOV-2855: add feature flag for fieldOtherVaLocations

aurora-a-k-a-lightning committed:
Merge branch 'master' of https://github.com/department-of-veterans-affairs/vets-website into vagov-2855

aurora-a-k-a-lightning committed:
VAGOV-2855: fix linting

aurora-a-k-a-lightning committed:
VAGOV-2855: fix linting

aurora-a-k-a-lightning committed:
VAGOV-2855: fix linting

ethanteague reviewed (approved):
LGTM :+1: @ahay-agile6

aurora-a-k-a-lightning committed:
Merge branch 'master' of https://github.com/department-of-veterans-affairs/vets-website into vagov-2855

jbalboni commented on src/site/layouts/health_care_region_locations_page.drupal.liquid:
```diff
@@ -19,6 +19,11 @@
+
+                    {% if fieldOtherVaLocations != empty %}
+                        <h2 id="community-clinic-locations">Other nearby VA locations</h2>
+                        <div class="cta-widget" data-widget-type="other-facility-locations-list" data-facilities="{{ fieldOtherVaLocations }}"></div>
```
Can we remove the cta-widget class from all these?

jbalboni commented on src/applications/static-pages/facilities/otherFacilityList.js:
```diff
@@ -0,0 +1,24 @@
+      default: OtherFacilityListWidget,
+    } = await import(/* webpackChunkName: "facility-locations-list" */ './OtherFacilityListWidget');
+
+    // since these widgets are on content pages, we don't want to focus on them
```
Can remove this comment

jbalboni reviewed (approved):
Couple minor changes, but LGTM otherwise

aurora-a-k-a-lightning committed:
Merge branch 'master' of https://github.com/department-of-veterans-affairs/vets-website into vagov-2855

aurora-a-k-a-lightning committed:
VAGOV-2855: review changes

+ removed `class="cta-widget"` from facility list react component elements
+ removed unnecessary comment
